test/check-machines-create

```diff
@@ -275,6 +288,62 @@ class TestMachinesCreate(VirtualMachinesCase):
+                                                                    os_short_id=config.FEDORA_28_SHORTID,
+                                                                    user_password="catsaremybestfr13nds",
+                                                                    user_login="foo",
+                                                                    root_password="dogsaremybestfr13nds",
+                                                                    ssh_keys=[rsakey, dsakey],
+                                                                    create_and_run=True))
+
+        # try to input multiple keys (separated by newline) into one text input, expect only first one to be used
+        runner.createCloudBaseImageTest(TestMachinesCreate.VmDialog(self, sourceType='cloud',
+                                                                    storage_size=10, storage_size_unit='MiB',
+                                                                    location=config.VALID_DISK_IMAGE_PATH,
+                                                                    os_name=config.FEDORA_28,
+                                                                    os_short_id=config.FEDORA_28_SHORTID,
+                                                                    user_password="catsaremybestfr13nds",
+                                                                    user_login="foo",
+                                                                    root_password="dogsaremybestfr13nds",
+                                                                    ssh_keys=[f"{rsakey} \n {dsakey}"],
+                                                                    expected_ssh_keys=[rsakey],
+                                                                    create_and_run=True))
+
+        # Try to create VM with one SSH key
+        runner.createCloudBaseImageTest(TestMachinesCreate.VmDialog(self, sourceType='cloud',
+                                                                    storage_size=10, storage_size_unit='MiB',
+                                                                    location=config.VALID_DISK_IMAGE_PATH,
+                                                                    os_name=config.FEDORA_28,
+                                                                    os_short_id=config.FEDORA_28_SHORTID,
+                                                                    user_password="catsaremybestfr13nds",
+                                                                    user_login="foo",
+                                                                    root_password="dogsaremybestfr13nds",
+                                                                    ssh_keys=["non-valid ssh key"],
+                                                                    ssh_key_invalid=True,
+                                                                    create_and_run=True))
```

Comment:
I think the one case that is missing is to add a key but no user name?

Reply:
It's already there: https://github.com/cockpit-project/cockpit-machines/pull/1150/files#diff-b284ac42752f40597c4dd12d74e16f6ed82ad5e12d68f0437f242f5ae20bbaa5R188